src/axom/inlet/examples/nested_structs.cpp

```diff
@@ -333,9 +344,13 @@ int main(int argc, char** argv)
 
   if(docsEnabled)
   {
-    std::unique_ptr<inlet::SphinxWriter> writer(
+    std::unique_ptr<inlet::SphinxWriter> sphinxWriter(
       new inlet::SphinxWriter("nested_structs.rst"));
-    inlet.registerWriter(std::move(writer));
+    inlet.registerWriter(std::move(sphinxWriter));
+    inlet.writeDoc();
+    std::unique_ptr<inlet::JSONSchemaWriter> schemaWriter(
+      new inlet::JSONSchemaWriter("nested_structs.json"));
+    inlet.registerWriter(std::move(schemaWriter));
     inlet.writeDoc();
```

Comment:
I wonder if `Writer::write()` is a more appropriate name now that it is not strictly a writer for docs.